pkg/config/config.go

```diff
@@ -0,0 +1,625 @@
+package config
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"sync"
+	"time"
+
+	"github.com/calyptia/api/client"
+	cloud "github.com/calyptia/api/types"
+	"github.com/calyptia/cli/pkg/formatters"
+	fluentbitconfig "github.com/calyptia/go-fluentbit-config"
+	"github.com/hako/durafmt"
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+)
+
+type Config struct {
+	Ctx          context.Context
+	BaseURL      string
+	Cloud        *client.Client
+	ProjectToken string
+	ProjectID    string
```

Comment:
I though the `config` package would have the `calyptia create` subcommands.
This `Config` struct holds shared flags and some dependencies used by the commands.

Reply:
This is unrelated to the commands. All commands will go into cmd package